spec/daemon_spec.rb

```diff
@@ -2,45 +2,99 @@
 describe ServerEngine::Daemon do
   include_context 'test server and worker'
 
-  it 'run and graceful stop' do
-    dm = Daemon.new(TestServer, TestWorker, daemonize: true, pid_path: "tmp/pid")
-    dm.main
+  unless ServerEngine.windows?
```

Comment:
This patch clearly describes that "We can NEVER support interprocess signal system on Windows".
I think `pending` means that "this is not supported now, but will be able to support some day".
Therefore, `pending` is not suitable for this test.